internal/pikamon/commands/catch.go

```diff
@@ -0,0 +1,123 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/Jac0bDeal/pikamon/internal/pikamon/util"
+	"github.com/bwmarrin/discordgo"
+	"github.com/mtslzr/pokeapi-go"
+	log "github.com/sirupsen/logrus"
+	"math/rand"
+	"strconv"
+	"strings"
+)
+
+var pokemonExpiredMessages = []string{
+	"The pokemon heard you coming and ran for the hills!",
+	"The pokemon thought you were ugly and didn't want to be caught by an ugly person!",
+	"The pokemon likes Physics (mistakenly) and thought you were too dumb to understand it so it ran!",
+}
+
+var catchFailureMessages = []string{
+	"You may need to have your eyes checked!",
+	"Better luck next time!",
+	"Have you tried getting good?",
+}
+
+func pokemonExpiredFailure(s *discordgo.Session, m *discordgo.MessageCreate) bool {
```

Comment:
all of the functions here that send to discord are better off returning errors to the caller, and should have consistent naming. I like what you did with the `publishSuccessfulCatch` name, so I'd suggest that naming convention.

Reply:
done